modules/network/src/test/java/org/apache/ignite/network/internal/AllTypesMessageGenerator.java

```diff
@@ -81,139 +79,142 @@ public static AllTypesMessage generate(long seed, boolean nestedMsg) {
-                }
-                return shorts;
-
-            case INT_ARR:
-                int intArrLen = random.nextInt(1024);
-                int[] ints = new int[1024];
-                for (int i = 0; i < intArrLen; i++) {
-                    ints[i] = random.nextInt();
-                }
-                return ints;
-
-            case LONG_ARR:
-                int longArrLen = random.nextInt(1024);
-                long[] longs = new long[1024];
-                for (int i = 0; i < longArrLen; i++) {
-                    longs[i] = random.nextLong();
-                }
-                return longs;
+    @Nullable
+    private static Object randomValue(Random random, Field field, boolean nestedMsg) {
+        Class<?> type = field.getType();
 
-            case FLOAT_ARR:
-                int floatArrLen = random.nextInt(1024);
-                float[] floats = new float[1024];
-                for (int i = 0; i < floatArrLen; i++) {
-                    floats[i] = random.nextFloat();
-                }
-                return floats;
-
-            case DOUBLE_ARR:
-                int doubleArrLen = random.nextInt(1024);
-                double[] doubles = new double[1024];
-                for (int i = 0; i < doubleArrLen; i++) {
-                    doubles[i] = random.nextDouble();
-                }
-                return doubles;
-
-            case CHAR_ARR:
-                int charArrLen = random.nextInt(1024);
-                char[] chars = new char[1024];
-                for (int i = 0; i < charArrLen; i++) {
-                    chars[i] = (char) random.nextInt();
-                }
-                return chars;
-
-            case BOOLEAN_ARR:
-                int booleanArrLen = random.nextInt(1024);
-                boolean[] booleans = new boolean[1024];
-                for (int i = 0; i < booleanArrLen; i++) {
-                    booleans[i] = random.nextBoolean();
-                }
-                return booleans;
-
-            case STRING:
-                int aLetter = 'a';
-                int strLen = random.nextInt(1024);
-                StringBuilder sb = new StringBuilder();
-                for (int i = 0; i < strLen; i++) {
-                    int letter = aLetter + random.nextInt(26);
-                    sb.append(letter);
-                }
-                return sb.toString();
-
-            case BIT_SET:
-                BitSet set = new BitSet();
-                int setLen = random.nextInt(10);
-                for (int i = 0; i < setLen; i++) {
-                    if (random.nextBoolean()) {
-                        set.set(i);
-                    }
-                }
-
-                return set;
-
-            case UUID:
-                byte[] uuidBytes = new byte[16];
-                random.nextBytes(uuidBytes);
-                return UUID.nameUUIDFromBytes(uuidBytes);
+        if (type == byte.class) {
+            return (byte) random.nextInt();
+        }
```

Comment:
Too many braces

Reply:
Is this rule obligatory in ignite 3? I can see both styles at the moment

Reply:
I think we prefer to use the same code style as before. Frankly, I don't remember such discussion so it's hard for me to convince you